### PR TITLE
Changes crossbow damage type

### PR DESCRIPTION
**Pull request by flpfs**
https://forums.aurorastation.org/topic/14952-energy-crossbow-lethality/

Toxin damage was too OP due to brainmed, it now only deals stun.

### CODE REVIEW

**Comment by flpfs** [comment]
!review

**Review by fernerr — COMMENTED** [review]
The description needs to be updated to match.
https://github.com/Aurorastation/Aurora.3/blob/307c21454159ce2e91b1d2e71abd2db260574136/code/modules/projectiles/guns/energy/stun.dm#L40

File: code/modules/projectiles/projectile/energy.dm [hunk 1]
```diff
@@ -78,17 +78,15 @@
...
 	stutter = 10
 
-
 /obj/item/projectile/energy/bolt/large
 	name = "largebolt"
-	damage = 20
-
+	damage_type = PAIN
+	agony = 60
+	stutter = 10
```
Comment (fernerr) [inline code/modules/projectiles/projectile/energy.dm:89]:
```suggestion
```
Inherited from the parent, so no need to declare it here again.

**Review by NonQueueingMatt — APPROVED** [review]
Not sure

**Review by Geevies — APPROVED** [review]

**Pull request merged**